src/create-app.ts

```diff
@@ -115,6 +115,9 @@ async function prepare(starter: Starter) {
       files: [join(filePath, '*'), join(filePath, 'src/*')],
       from: /stencil-starter-project-name/g,
       to: projectName,
+      glob: {
+        windowsPathsNoEscape: true,
+      },
```

Comment:
@tanner-reits || @christian-bromann can you offer a second set of eyes on this PR? I had to update this in order to get CI to pass on Windows. See 0e46aa78145d241358cb4bc5f3abc74b13eae712 for more information